ffs/auth/auth.go

```diff
@@ -27,8 +27,8 @@ type Auth struct {
 }
 
 type entry struct {
-	Token      string
-	InstanceID ffs.APIID
```

Comment:
What about just `FSID` (_file system ID_)?

Reply:
So, changing _API_ for _FS_? (checking if understood correctly).
Sounds coherent with FFS. A particular instance of the _Filecoin File System_ (FFS) is a _File System_ (FS), plus the name is short which is nice. 

Opinions? @asutula @andrewxhill

Reply:
sounds reasonable, if it matches your intuition i trust it

Reply:
Cool, let's see @asutula opinion and try to take some decision!

Reply:
Hmm, none of those sounds obvious to me, but I also don't have any great idea. Instead of getting all wrapped up in our own names and architecture, we could look at it more from the user perspective... In that way, it's like a session. You create an FFS Session and you get a Session Token. I know that doesn't map as nicely to the actual code and architecture, but it makes sense.

Reply:
Uhm, Session sounds like a ephemeral thing without identity. 
I'll incubate that idea and see if I can thing in some more options. 

For some reason I keep coming to the Instance name again. A client has an FFS instance...